tools/RAiDER/models/hrrr.py

```diff
@@ -134,152 +141,123 @@ def load_weather(self, *args, filename=None, **kwargs):
         self._lons = _lons
 
 
-    def _makeDataCubes(self, filename, out=None):
-        '''
-        Create a cube of data representing temperature and relative humidity
-        at specified pressure levels
-        '''
-        if out is None:
-            out = filename
-
-        # Get profile information from gdal
-        prof = rio_profile(str(filename))
-
-        # Now get bounds
-        S, N, W, E = self._ll_bounds
+class HRRRAK(WeatherModel):
+    def __init__(self):
+        # The HRRR-AK model has a few different parameters than HRRR-CONUS. 
+        # These will get used if a user requests a bounding box in Alaska
+        super().__init__()
+        self._classname = 'hrrrak'
+        self._dataset = 'hrrrak'
+        self._Name = "HRRR-AK"
+        self._time_res = TIME_RES['HRRR']
+        self._valid_range = (datetime.datetime(2018, 7, 13), "Present")
+        self._lag_time = datetime.timedelta(hours=3)
+        self._valid_bounds =  Polygon(((195, 40), (157, 55), (175, 70), (260, 77), (232, 52)))
+
+        # This projection information will never get used but I'm keeping it for reference
+        # for the HRRR-AK model. The projection information gets read directly from the 
+        # weather model file. 
+        # self._proj = CRS.from_string(
+        #     '+proj=stere +ellps=sphere +a=6371229.0 +b=6371229.0 +lat_0=90 +lon_0=225.0 ' +
+        #     '+x_0=0.0 +y_0=0.0 +lat_ts=60.0 +no_defs +type=crs'
+        # )
+    def _fetch(self):
+        pass
+    def load_weather(self, *args, **kwargs):
+        return super().load_weather(*args, **kwargs)
+
+
+def download_hrrr_file(ll_bounds, DATE, out, model='hrrr', product='prs', fxx=0, verbose=False):
+    '''
+    Download a HRRR weather model using Herbie
+
+    Args: 
+        DATE (Python datetime)  - Datetime as a Python datetime. Herbie will automatically return the closest valid time,
```

Comment:
@dbekaert if I understand your comment correctly this is what we do for the GUNW workflow. The code will request specifically the two times exactly available on either side of the requested time, so that there shouldn't ever be an issue with inconsistency.